invenio_pages/__init__.py

```diff
@@ -1,12 +1,50 @@
 # -*- coding: utf-8 -*-
 #
 # This file is part of Invenio.
-# Copyright (C) 2015-2022 CERN.
+# Copyright (C) 2015-2023 CERN.
 #
 # Invenio is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
-"""Static pages module for Invenio."""
+r"""Static pages for Invenio.
+
+Invenio-Pages provides means to add static pages to Invenio. 
+Once the module is installed, developers will be able to customise
+their Invenio instance by registering custom static pages in a 
+``pages.yaml`` file.
+Define static pages
+----------
+To add static pages, the developer must create (if it doesn’t exist) a 
+pages.yaml file inside the app_data folder of their instance folder.
+.. code-block:: python
+    <your_site>/
+    └──app_data/
+        └── pages.yaml
+Once the file is created the developer has to define in it the static 
+pages it wishes to create, using the following format for each page:
```

Comment:
`pages it wishes to create, using the following format for each page:`
is `it` referring to the developer? 
If so shouldn't it be `they`?